### DISCUSSION TITLE
I submitted the documents, but it doesn't look like anything's happening

### DISCUSSION THREAD

**Discussion by Saryrndeth** [community/community#72822] [GitHub Education]
### Hi everyone,

I submitted a total of 4 documents in Korean and English, but nothing happened without the statement that it was rejected. Is my approval in progress?

video: https://youtu.be/KfoXtMx5EDE

**Comment by saty-a** [comment]
It seems like you've submitted four documents in both Korean and English, but you haven't received any notification about their status, neither an approval nor a rejection. If you haven't received any feedback, it's possible that your documents are still under review. The absence of a rejection notice could indicate that they're being considered or that the review process might be taking longer than anticipated. It took almost **2 weeks** for my approval on the mail.
